tsconfig.json

```diff
@@ -5,6 +5,8 @@
     /* Language and Environment */
     "target": "es2022", 
     "lib": ["dom", "es2022"],
+    "jsx": "react-jsx",
+    "jsxImportSource": "preact",
```

Comment:
Added these options under `Language and Environment`, as that's where they appear in https://aka.ms/tsconfig